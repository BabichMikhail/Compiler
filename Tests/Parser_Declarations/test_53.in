var
	a: record
		a: record 
			a: record
			end;
		end;
		b: record 
			a : record
			end;
		end 
	end;