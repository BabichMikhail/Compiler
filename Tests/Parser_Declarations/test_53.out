Var   a
   Record
      Var   a
         Record
            Var   a
               Record
               End

         End

      Var   b
         Record
            Var   a
               Record
               End

         End

   End

